oci/pack.go

```diff
@@ -809,6 +819,9 @@ func (ocipack *ociPackage) Unpack(ctx context.Context, dir string) error {
 		}
 	}
 
+	// Set the environment variables
+	ocipack.Metadata().(*ocispec.Image).Config.Env = image.Config.Env
```

Comment:
Since we're within the `oci` package, why not directly set the value rather than via indirection + casting?
```go
ocipack.config.Config.Env = image.Config.Env
```
?